resources/js/features/user-game-activity/components/UserGameActivityTimeline/HashLabel/HashLabel.tsx

```diff
@@ -0,0 +1,74 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { LuDisc3 } from 'react-icons/lu';
+
+import {
+  BaseTooltip,
+  BaseTooltipContent,
+  BaseTooltipTrigger,
+} from '@/common/components/+vendor/BaseTooltip';
+
+interface HashLabelProps {
+  session: App.Platform.Data.PlayerGameActivitySession;
+}
+
+export const HashLabel: FC<HashLabelProps> = ({ session }) => {
+  const { t } = useTranslation();
+
+  if (!session.gameHash) {
+    return (
+      <BaseTooltip>
+        <BaseTooltipTrigger>
+          <span>{t('Unknown Hash')}</span>
+        </BaseTooltipTrigger>
+
+        <BaseTooltipContent className="max-w-[280px]">
+          <p className="text-xs">
+            {t(
+              "Either the user's emulator isn't reporting what hash they're using, the user's session was created before we started recording hashes, the user closed the game very quickly, or the user's hash was later unlinked from the game.",
+            )}
+          </p>
+        </BaseTooltipContent>
+      </BaseTooltip>
+    );
+  }
+
+  if (!session.gameHash.name) {
+    return <span>{session.gameHash.md5}</span>;
+  }
+
+  return (
+    <div className="flex gap-2">
+      {session.gameHash.isMultiDisc ? (
+        <BaseTooltip>
+          <BaseTooltipTrigger>
+            <LuDisc3 className="size-4 text-cyan-300" />
```

Comment:
Why is the disc colored? It feels out of place with its surroundings.

Reply:
Removed special coloring for that icon in latest.